app/src/main/java/ro/code4/monitorizarevot/ui/base/BaseActivity.kt

```diff
@@ -131,4 +132,14 @@ abstract class BaseActivity<out T : BaseViewModel> : AppCompatActivity(), Layout
             .show()
     }
 
+    //    Collapse the keyboard when the user taps outside the EditText
+    override fun dispatchTouchEvent(motionEvent: MotionEvent): Boolean {
+
+        currentFocus?.let { oldFocus ->
+            super.dispatchTouchEvent(motionEvent)
+            val newFocus = currentFocus ?: oldFocus
+            collapseKeyboardIfFocusOutsideEditText(motionEvent, oldFocus, newFocus)
```

Comment:
It's better to return result from line 139 here instead of calling the super method again

Reply:
good catch! we'll have to fix this in a new PR